### DISCUSSION TITLE
Request for mod use approval

### DISCUSSION THREAD

**Discussion by taintedvalor** [FTBTeam/FTB-Mods-Issues#952] [General]
So I've attempted to reach out in several ways. There doesn't appear to be an official way to request approval or a statement denying it explicitly, so I'm attempting to reach out here. I have a small Minecraft server on fabric 1.19.2, and I have a custom mod pack. For convenience, I host it on modrith, which requires explicit approval to host a modpack with any mods under 'all rights reserved.'  Since there doesn't appear to be any statements regarding modpack use, I am reaching out here. I would appreciate a timely response. Thank you.

**Comment by MichaelHillcox** [comment]
At this time, we are not providing permission for modrinth to rehost our mods. Sorry about that.
